### ISSUE TITLE
<CPAP Supply Reorderingt> from <Health Portfolio>

### ISSUE THREAD

**Issue by JasmineYohannan** [department-of-veterans-affairs/va.gov-team#61624]
## We are a team focused on adding CPAP machines to the hearing aid and batteries tool by the end of summer 2023. Following this we will be looking to add other features/supplies to this tool. 

- Team name: 
-	OCTO-DE product owner: Patrick Bateman
-	Product manager: Jasmine Yohannan
-	Designer: 
-	FE engineer: 
- Accessibility specialist:
- Content writer: 
-	Product/team Slack channel: va-cto-supply-reordering
 
### What does your team need support for? Check all that apply.TH 
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [X] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [X] Something else 
 
### Give a brief description of what your team needs support with:

My team needs support with: 1. How to update the H1s and any other content to accommodate adding CPAP machines to the hearing aid and batteries reorder tool. 2. This is after initial launch, but we need to decide if we need to break apart prosthetic socks with the growing universe of supplies that can be ordered online
 
### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [X] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*

### When does this work need to be completed?
- Estimated launch date: End of August 2023
- Estimated staging review date: N/A
- Content, Accessibility, and IA work needed by:  August 2023

### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [X] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- Ticket (number): 
- Collab. Cycle Ticket Opened (date): 
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [X] Accessibility
- [X ] Content
- [X ] IA
- [X] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Midpoint
- [] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [X] Unknown at this time.
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [X] Unknown at this time.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows: --
- Link to prototype or mockups:--
- Link to your research plan: --
- Link to any epics/issues that may be helpful:

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by laurwill** [comment]
Hi @JasmineYohannan ! A couple questions as we set up our CAIA work for this request:
- Do you expect to add more supplies to the online ordering system in the future (and is there a list of those supplies)?
- Are prosthetic socks ever going to move into the online system, or will those stay on a paper form for the foreseeable future?

Thank you!

**Comment by JasmineYohannan** [comment]
Hi @laurwill thanks for your reply. To answer your questions:

- We do expect to add more supplies, but there is no list at the moment,and it is out of scope for this ticket as we are primarily focused on solely adding CPAP supplies as that's the most immediate need.
- Prosthetic socks will stay on a paper form for the foreseeable future.

**Comment by kristinoletmuskat** [comment]
Hey ya'll, thanks for submitting this ticket! Here are some high-level IA thoughts to consider:

I wonder if a separate static page for CPAP could be helpful (instead of combining it with the hearing aids page), given the amount of content on the hearing aids static page. Having separate pages might also help the SEO of those two very different things. I’m not as concerned about the amount of questions on the form, it’s just kind of weird to have 1 form with those 2 very diff things.

But, I also wanted to link to this [thread for situational awareness](https://dsva.slack.com/archives/C52CL1PKQ/p1689692133372719) about multiple sign-in widgets on one page. We are planning to talk through this with Danielle next week, but essentially when someone is logged in I think we’re trying to redirect them to the logged-in version of the page? I’m not quite sure, but it seems like adding a separate login widget for a separate CPAP form might not be the way to go because of that.

If we were to consider adding a new page, we could go with one of these methods -- either creating some sort of 'order supplies' group with the different supplies as separate pages, or adding CPAP as a new page for now. I think once we have 4+ supply types, we will want some sort of additional nav level.
![Screenshot 2023-07-24 at 4 39 58 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/db50f63a-6f75-4a85-87cc-265d0036ef37)
![Screenshot 2023-07-24 at 4 40 05 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/9bc1a98f-3793-41b4-84d8-414396598a5f)

Thoughts about this? @batemapf @JasmineYohannan

**Comment by eli-oat** [comment]
Hey ya'll, I wanted to check in on this ticket -- it is noted that you won't be going through the Collaboration cycle, but then it is checked that you'd like assistance with accessibility during the Design Intent step of the Collab Cycle. 

When you have a second, could you clarify this so that we can determine the best next steps to getting you the accessibility support that makes the most sense? 

Thanks!

**Comment by JasmineYohannan** [comment]
@eli-oat Hi! This might be a user error on my end, this is my first time filling out a Content/IA ticket, sorry for the inconvenience! We are definitely not doing the collab cycle for this work, but I wanted to have someone help us with the Content, IA, and Accessibility for any CPAP content we end up making for this. I can go ahead and uncheck those boxes if that makes it more clear.

**Comment by kristinoletmuskat** [comment]
HI ya'll, I know we talked @JasmineYohannan and @batemapf in slack about the IA decision. It's also documented in the IA spec doc here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/order-cpap.md

lmk if anything looks funky -- I was having some weird commit issues.

**Comment by JasmineYohannan** [comment]
@kristinoletmuskat Everything looks good on the spec doc to me! cc: @batemapf

**Comment by laurwill** [comment]
Hello @JasmineYohannan , thanks for the ping on Slack — following up on the next steps for content development here. I've listed each one here with questions.

### New static page in the benefit hub: "Order CPAP supplies"
- To draft this static page content, we'll need all the relevant information about CPAP supplies. Are you already working with the partners and SMEs who can answer questions we have about this content, and approve the page when it's ready? 
- For example, I found this [VA medical center page](https://www.va.gov/tampa-health-care/programs/cpap-ordering-distribution-and-supply-information-sheet-and-faqs/) that mentions recent CPAP recalls and supply shortages — we'll want to confirm whether we need Q&As on our static page to address those types of questions.
- We'll also need to confirm whether eligibility requirements and other information on the [hearing aid page](https://www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/) applies to CPAP ordering. For example, does DLC repair CPAP devices? Is there a DLC catalog of CPAP supplies, similar to the linked batteries catalog on the hearing aid page? 
- We can either send a list of questions up front, or draft as much of the page as possible with current information and send a draft for review with remaining questions.

### Changes to online form to include CPAP supplies
- Do you have in-process designs or a content source of truth for the online form? 

### Changes to PDF form and About this form page in forms library
- Is your team planning to replace the 2 current PDF forms [2346a](https://www.va.gov/find-forms/about-form-2346a/) and [2346b](https://www.va.gov/find-forms/about-form-2346b/) with a new combined form for hearing aid and CPAP supplies?
- Will your team be working with the stakeholders to get approvals to deprecate the current PDF forms and launch the new combined PDF form?
- @kristinoletmuskat if this is happening, we'll also need redirects for the current "About this form" pages to the new combined About this form page.

**Comment by laurwill** [comment]
Oh and @JasmineYohannan one more note here — you commented above that this won't be going through the collaboration cycle. In the past, we've been required to bring any new static page in a main benefit hub to a staging review only (not design intent or midpoint). This page would fall in that category.

**Comment by JasmineYohannan** [comment]
@laurwill thank you for outlining all of this!

To answer your questions - Yes we have contacts with DLC, and other SMEs who can answer any questions we may give them about content. I think it would be best for the sake of timeline to draft as much of the page as possible and send any remaining questions we have. Do you want me to pass along the two you've listed here? (supply shortages + eligibility requirements/repairs).

In terms of the online form -- we already have the functionality in place/showing for ordering CPAP supplies in testing environments, could you clarify the "content source of truth"?

As far as the Forms question -- Creating a new combined PDF would be out of scope for MVP in September. I think we should stick to linking to Form 2346b for now. cc: @batemapf

**Comment by laurwill** [comment]
Hi @JasmineYohannan , 
- Ah that's my mistake, I didn't realize the form changes were already in testing environment — is there a prototype or a specific test user info we need to see this in staging? And could you link to the most recent designs for the form changes in Sketch (or wherever they live?) We want to make sure our page content matches how the online form presents the CPAP options.
- Let's hold on sending questions while we start drafting — we can get you a more complete list on Monday. We can proceed with drafting while we wait for a reply.
- Are there any documents or additional info the SMEs have shared with you about how CPAP ordering works that we can refer to as we draft the page? 
- And we'll move forward with linking the existing 2346b form for now. Thank you!

**Comment by laurwill** [comment]
Thanks @batemapf ! This is great. Couple questions for you:
- Could you also screenshot content for "What if the sleep apnea supplies I need aren't listed here?"
- Did you choose "sleep apnea" instead of "CPAP" for the H1 based on VA site analytics or other SEO data? 

And here's a list of our questions as we're drafting the page content — from the screenshots, it looks like you may already have answers to some of these.

1. Are these the correct eligibility requirements for CPAP supplies?
- You’re enrolled in VA health care, and
- You’re registered as a patient at a VA medical center, and
- Your VA provider has prescribed a CPAP machine
2. Can you only order a supply online if you've ordered that same exact supply in the past 2 years? If you need a different/new supply, do you use the PDF form for the first order?
3. We found this [local VAMC page about delays and recalls related to CPAP machines from 2022](https://www.va.gov/tampa-health-care/programs/cpap-ordering-distribution-and-supply-information-sheet-and-faqs/) — does that 6 to 8 week timing for supply orders still apply here? Is there any other current information related to recalls or supply shortages that would affect this ordering process?
4. What should they do if their CPAP machine isn't working? Does VA provide any central CPAP machine repair service (similar to the hearing aid repair service), or should they contact their local facility?

**Comment by JasmineYohannan** [comment]
@laurwill I can answer some but not all of these questions, will report back when I have the rest:

- I don't think we have screenshots for that content but it would be pretty similar to the hearing aids text "The accessories you need may not be listed here if you haven’t placed an order for resupply items within the last 2 years. If you need sleep apnea supplies that hasn’t been ordered within the last 2 years, call the DLC Customer Service Section at 303-273-6200 or emaildalc.css@va.gov." I will reach out to DLC to see if that is the best number to reach or if there is a separate one for hearing aids and update here. 


- We chose sleep apnea after talking to the Denver Logistics Center and other SMEs who are more familiar with this work, they've found that Veterans most commonly use "sleep apnea supplies" or "CPAP resupplies"
- Those are the correct eligibility requirements for re-ordering CPAP supplies, yes
- For questions 2-4, I will need to reach out to DLC folks but tagging @batemapf if he has any additional info on this until I am able to get a response and update here

**Comment by laurwill** [comment]
Thanks @JasmineYohannan ! That's very helpful. 
From a quick keyword search, "CPAP supplies" seems much more commonly used. I can do a bit more research next week, but based on these numbers we'll probably suggest using "CPAP" in the H1s instead of "sleep apnea." Based on your conversations with SMEs, do you think that change would make sense?

VA site search in the past year:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/55a6c1d0-dcfc-4baa-9adb-c242722d060b)
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/c44fa354-9150-46a5-b631-a48023b1bbbe)

SEMrush keyword volume:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/81b41f11-4998-4020-ad08-70a67a1e7c5d)
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/09c85c1e-6283-4a71-90b7-fc306d35f197)


And cc @d-beyer , for the new info Jasmine provided. Could you update the page draft as needed? Thanks!

**Comment by d-beyer** [comment]
@laurwill I've made adjustments to the draft.

**Comment by JasmineYohannan** [comment]
@laurwill just confirming here that "CPAP supplies" is good for the h1! Still waiting on a response for the other questions

**Comment by laurwill** [comment]
Hi @JasmineYohannan , yes, we should make that change consistently throughout the tool. Are the tool designs in Sketch, or somewhere else where we could easily leave comments on them with content suggestions?

**Comment by JasmineYohannan** [comment]
Hi @laurwill just got a response back from DLC on questions 2-4, here are their responses:

-  Can you only order a supply online if you've ordered that same exact supply in the past 2 years? If you need a different/new supply, do you use the PDF form for the first order

**You can only reorder the same item that had been previously ordered.  Veterans should contact their clinician, it they need other items**

- We found this [local VAMC page about delays and recalls related to CPAP machines from 2022](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.va.gov%2Ftampa-health-care%2Fprograms%2Fcpap-ordering-distribution-and-supply-information-sheet-and-faqs%2F&data=05%7C01%7C%7C943d8864e54c419d268908dba9649a89%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C638290021778127332%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tCR1j%2FERcuGkExpRh2cNGNsKf%2FiwfZt7WMC5UcxRpeA%3D&reserved=0) — does that 6 to 8 week timing for supply orders still apply here? Is there any other current information related to recalls or supply shortages that would affect this ordering process?

**No. Veterans will receive resupplies within 7-10 days.**

- What should Veterans do if their CPAP machine isn't working? Does VA provide any central CPAP machine repair service (similar to the hearing aid repair service), or should they contact their local facility?

**They should contact their local facility or customer service. Their Provider will order a new machine for the Veterans.**

**Comment by JasmineYohannan** [comment]
@laurwill @d-beyer Would it be possible for me to access/take a look at the draft? Thanks!

**Comment by laurwill** [comment]
Hi @JasmineYohannan , yes! Here's the [page draft with remaining questions](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694016075657/81eac28399c405179a1f8bb4d564de153350d7c3?sender=u3038037989bec1f893349894) in sharepoint — could you see if you know any of those answers before we send to SMEs? 

We'll also plan to leave comments with content suggestions on the form [screenshots in this Mural ](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694016075657/81eac28399c405179a1f8bb4d564de153350d7c3?sender=u3038037989bec1f893349894)as well. 

Would you like us to send the page draft for SME review now, or wait until we've reviewed the form content? We'll want to explain to SMEs that we're asking for an accuracy review only and that the page content needs to follow our VA.gov style and voice, etc. Also happy to join a call with the SMEs, if that would be helpful.

**Comment by kristinoletmuskat** [comment]
Hey ya'll, after chatting with Laura and catching up with this thread, I updated the IA spec doc to include the new H1 we've decided on for the statis CPAP page: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/order-cpap.md

Laura also flagged this Q -- should we change the hearing aid form url since we're adding CPAP? I think we should if we can and there aren't huge technical constraints. I added a redirect section to the bottom of the IA spec documenting the new URL. For these, ya'll will have to submit a [redirect ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request) at least 2 weeks in advance of the form launching.

**Comment by JasmineYohannan** [comment]
@laurwill The page draft link leads me to a mural board. The only question I could see was about adding "if applicable" to the" hearing aid information" to the prepare step - I definitely think we should do that.

I do have another question for you all - the DLC team asked if there is a difference between having the tool say "Order hearing aid and CPAP supplies" vs. "Order hearing aid or CPAP supplies" , and which you all think is better to stick with for this tool?

**Comment by laurwill** [comment]
Hi @JasmineYohannan , so sorry about that. Here is the correct link to the [Sharepoint document](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/EWWA8N14uW5AhhWdKSAfx9wBht4gc2ztk_RnTllOK5ssDA?e=jH5jjo)!

**Comment by laurwill** [comment]
Hi @JasmineYohannan ,
Just confirming that we've added content notes to the [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694016075657/81eac28399c405179a1f8bb4d564de153350d7c3?sender=u3038037989bec1f893349894).

And could you confirm that the online form changes are **not** going through collab cycle? Normally we'd make the new static "order CPAP supplies" entry page part of the staging review, so trying to figure out if we need a separate staging review just for the static page.

**Comment by JasmineYohannan** [comment]
Hi @laurwill we will not be going through the collab cycle for the form changes.

**Comment by laurwill** [comment]
hi @Jasmine Yohannan, thanks for confirming — since your team isn't bringing the online form stages through collab cycle, we'll need to schedule a single-step staging review for just the static page. Our team can request that review once we know when the content will be final.

- I think collab cycle will need the link from our page to the online form to be working, so they can validate the destination. 
- We can move ahead with building the page in Drupal. We still have some questions to confirm with the SMEs about the page content. We also had a couple questions called out in the form Mural. Would you like us to get a package together to send to the SMEs with the draft page preview and remaining questions, or have you already shown them the page content?

**Comment by JasmineYohannan** [comment]
@laurwill I would like a package together to send to the SMES with the draft page preview and remaining questions, thanks!

What do you need from me for the first bullet? 

Also - can you give an estimate on when we will be able to push these changes into production?

**Comment by laurwill** [comment]
Hi @JasmineYohannan ! Our content team is OOO today for an offsite, but I can work on getting the SME materials together tomorrow. I'll also look into getting the staging review scheduled so we can have an estimated publishing date.

**Comment by batemapf** [comment]
👋 for planning/prep purposes, VHA will start marketing the new CPAP resupply functionality the week of 10/2.

**Comment by laurwill** [comment]
@batemapf I'm not sure the new static page will be live by then — what link will they be using to promote this? Will they be linking directly to the first page of the online form?

**Comment by batemapf** [comment]
we can tell them to link directly to the tool, yes. but i don't think they are going to feel comfortable moving forward if the full flow isn't in place (e.g., accessible from the health hub nav, shows up in va.gov search results, indexed by external search engines). 

is there a list of outstanding tasks that need to be completed before the static page and updated IA can go live?

**Comment by laurwill** [comment]
Hello @JasmineYohannan and @batemapf ,
I'm attaching a Word doc here that outlines the SME review request for the new static page along with our remaining content questions. I'm also attaching a screenshot of the draft page, in case SMEs have any trouble getting on the network. 

Have the SMEs already reviewed the online form content? I didn't include that here, because I wasn't sure if you'd made updates based on our notes in the Mural. 

Let me know if you'd like me to send these to SMEs directly. Also happy to join a call with SMEs if that would help move this along.

[SME review request for CPAP supplies page.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/12715463/SME.review.request.for.CPAP.supplies.page.docx)

[Order CPAP supplies page screenshot.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/12715480/Order.CPAP.supplies.page.screenshot.pdf)

Here's the list of steps I think we'll need before the page can go live. @kristinoletmuskat anything I'm missing here?
- Confirm URL for online form that will work during staging review (and URL to use in production, if different) — Health Portolio team
- Schedule staging review — CAIA team (waiting for response about process from [collab cycle in ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66173))
- Make any content updates based on SME review and staging review — CAIA team
- Make any needed content updates to online form — Health Portfolio team
- Final copyedit and prepublishing checklist — CAIA team
- Publish page and add to navigation — CAIA team

**Comment by JasmineYohannan** [comment]
@laurwill thank you for the detailed doc. I have sent it over to the SMEs, and should have answers and feedback relatively quickly. I can pass their answers along over email, and post them here as well.

**Comment by JasmineYohannan** [comment]
@batemapf can the changes from this [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694016075657/81eac28399c405179a1f8bb4d564de153350d7c3?sender=u3038037989bec1f893349894) to the form can be made at this point? Not all are mandatory - mostly redundant text removal/changing headings

**Comment by JasmineYohannan** [comment]
@laurwill 

Answers for remaining questions:

- Are CPAP orders also a 6-month supply, with orders limited to once every 5 months (same as hearing aids)? **For now, in the future we’re going to make it a one year supply.**

- The online hearing aid supplies form lists an email address for help: dalc.vss@va.gov. This email address doesn’t appear on the hearing aid supplies webpage. Is email a preferred way to contact the DLC? Can Veterans order supplies via email, or is this only for questions? **This is only for support.**

- Can you confirm if CPAP supplies are free of charge when you order through VA? **The DLC doesn’t bill any veterans for supplies. Veterans may have a co-pay at their Doctor visit.**

- We’ve listed the supplies from the PDF form in alphabetical order. Is there a more helpful order you’d suggest, such as frequency of orders for each item? **It looks good to me.**

- Is calling the facility the best way to get help with a broken CPAP machine? Or would sending a secure message directly to the provider be a better option? **Sending a secure message would be best if possible.**

- You mentioned they can also call DLC customer service if their machine isn’t working. Does the DLC provide repair or replacement services directly? Is one option (contacting VA provider vs DLC customer service) recommended over the other? **They should call the provider, but the DLC could contact the provider for the veteran, if needed.**

**Comment by JasmineYohannan** [comment]
@batemapf see the second bullet above - can you provide the credential?

@laurwill The confirmed URL is /health-care/order-hearing-aid-or-cpap-supplies-form

**Comment by batemapf** [comment]
for staging, use vets.gov.user+1 in [TUD](https://tud.vfs.va.gov/?page=1)

**Comment by JasmineYohannan** [comment]
@laurwill @kristinoletmuskat @d-beyer just want to check in on when we are going to publish the new static page? and should the redirect happen before or after that?

**Comment by laurwill** [comment]
Hi @JasmineYohannan ,
We're tracking on the redirect conversation in Slack. For now, we'll leave the link in the published static CPAP page as /health-care/order-hearing-aid-or-CPAP-supplies-form. We can update the link as needed.

We've also made draft changes to 3 related pages:
- [About VA Form 2346b, CPAP form page](http://preview-prod.vfs.va.gov/preview?nodeId=6186) — added "When to use" blurb and link to new online form. 
- [About VA Form 2346a, hearing aids form page](http://preview-prod.vfs.va.gov/preview?nodeId=6186) — added "When to use" blurb and replaced link to online form with new link
- [Order hearing aid batteries and accessories page](http://preview-prod.vfs.va.gov/preview?nodeId=36) 

We have one additional question and a note for the CPAP SMEs:
- **Question:** The PDF form uses PAP instead of CPAP. But CPAP has much higher search volume and seems to be the most commonly used term and type of PAP machine. Do SMEs see a need to add information about ordering other types of PAP supplies (APAP, BiPAP) to our pages? Or is CPAP the term Veterans are likely to use for any PAP-related machines and supplies? 
- **Note:** We'll follow up about changing the "call your facility" instruction to direct to secure messaging. We've had some other recent SME conversations on other health tool projects about how and when to direct to secure messaging instead of other contact options, and want to make sure we're doing this consistently across pages.

And here's a list of changes we made to the hearing aids page for consistency with new CPAP page:
- Updated hours for Denver Logistics Center phone support and put hours in ET
- Made the entry point to the online form a primary action link instead of the sign in widget, and directed the link to the new combined form
- Updated all other links on the page to follow current site style, placing links on a separate line instead of embedded in a sentence

Are the partners you're working with for the CPAP page also the group who would approve changes to the hearing aids ordering page? If not, we can help identify the correct group to approve those changes.

**Comment by kristinoletmuskat** [comment]
Hey @JasmineYohannan , just following up about your question in slack asking if we can add more entry points to the CPAP form.

Hub page: Each section in the hub page should have a max of 10 links. The manage section does have a link to the hearing aid page, but is already at 10. I wonder if we could link to the new joint form instead from the health hub (so replace the hearing aid link with the combined hearing aid/CPAP form link). 

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/512ea25e-4516-4df7-b282-131588fa8855)


Main Nav: this shows the top 5 links in the health hub, and it does currently list hearing aids. I wonder if we could do the same thing -- replace that link with the new joint form link:
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/122126772/3b600d65-2c97-4663-a581-4179783c116d)


@laurwill any thoughts about that?

**Comment by JasmineYohannan** [comment]
@kristinoletmuskat @laurwill I think we want to avoid this strategy because they have two different content pages with slightly different information.

Would it be possible to take a look at the analytics to see if it's the right call to potentially replace hearing aids with the CPAP link in the megamenu/health hub? And in the health hub page, just curious to know the why behind the 10-link limit?

Another option could be to create a general supplies page that links to both pages, and lists all supplies available to reorder whether it is through VA.gov or not and replace the hearing aid link with that in both places.

**Comment by JasmineYohannan** [comment]
@laurwill sorry for missing out on your earlier message. The same group from DLC would likely approve the hearing aid page changes as well. I can pass along the updates.

I can float the question about including information  specific to other -PAP machine supplies out to DLC as well, but my understanding is that most Veterans use CPAP as the umbrella term for all PAP-related supplies.

**Comment by laurwill** [comment]
No worries @JasmineYohannan , and thanks for coordinating SME review for these! 
- That's also our understanding, that CPAP is the umbrella term. Just wanted to confirm if there's any need for a note or mention of other types on the main page. 
- I'll pass these 3 updated pages (hearing aids page and 2 form detail pages) to our copyeditor for a final review, but we'll hold to publish until we hear back from you about SME approval. I'll be OOO next week, but you can tag @d-beyer with any updates about these pages.
- We'll also connect with Kristin about the linking questions (she's OOO today) and get back to you on that.

**Comment by laurwill** [comment]
Hi @JasmineYohannan ! Following up on the remaining page updates needed here — did you already send those to SMEs for an accuracy review? If not, I'd be happy to send them directly if you can confirm who needs to review. 

I'll be OOO from 12/18 to 1/2, and CAIA team will mostly be out for the last week of the year, so it would be great to wrap these up this week!

**Comment by JasmineYohannan** [comment]
@laurwill I did get an answer about the -PAP machines from our SMEs and they confirmed that "CPAP is the umbrella" term. You can email Patrick Booth - Patrick.Booth@va.gov directly to get approval for the hearing aid pages.

**Comment by laurwill** [comment]
Hi @kristinoletmuskat and @JasmineYohannan , we got SME approval to publish the related page updates, but the SME also asked about adding the CPAP page to the top nav menu. That list is already maxed out at 5 links, right Kristin?

So returning to the conversation about menu and hub links, I checked site search analytics for the past year, and hearing aids has more volume than CPAP. So I'm not sure we want to swap the hearing aids link for CPAP.

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/fbd7d1c5-60c3-4432-9573-2da1cc3febdb)

![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/104158104/b021fe71-d42c-43ff-abbb-a8c7ceb201e4)

**Comment by laurwill** [comment]
@JasmineYohannan confirming we published the related page updates to the hearing aids page and the form detail pages for both hearing aids and CPAP, after Patrick approved.

**Comment by kristinoletmuskat** [comment]
Hey hey yes @laurwill, @JasmineYohannan and I synced on the metrics in the past and made the same determination (to leave hearing aids at top and not add CPAP). The mega menu is limited to 5 items from get and 5 items from manage, and CPAP doesn't make the cut yet. We pulled month-long data in November and will continue to monitor: https://docs.google.com/spreadsheets/d/10vkw-_qm6cpo9C3bOCCl5KDcGRKwTO0PYvTSM0EUuKc/edit#gid=0

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mnorthuis** [comment]
I think the work needed on this is:
- Rework the page to accommodate the heading structure per #77499
- Widget placement on the Drupal landing page, and remove widget from /introduction page
- Move form into /my-health and behind authentication

**Comment by VCM86** [comment]
Update: I asked Sara about this, and she believes the Horizon team is still blocked by a server migration that ROES is going through. There is no current staging environment for the current version of this application, which the team refers to as the "Classic" version. The ROES team does not have test users in a lower environment, and has not since February 2025.
